docs/changes/16159.other.rst

```diff
@@ -0,0 +1,13 @@
+Fix loading parser tabs on pyc-only installations.
+
+Fix a bug in the wrappers for the lex and yacc wrappers that are
+used for parsing Astropy units so that they work on pyc-only
+installations.
+
+According to the Python module loading
+`flow chart <https://peps.python.org/pep-3147/#flow-chart>`_, when evaluating
+`import foo` and `foo.py` is not found, Python then reads `foo.pyc`.
```

Comment:
These all need double back ticks otherwise they get interpreted as links - this is why readthedocs fails.

Reply:
Fixed.